docs/manual/troubleshooting.tex

```diff
@@ -764,14 +764,28 @@ \section{Building from source\label{build-source}}
 \subsection{Install prerequisites\label{building-prerequisites}}
 
 You need to install several packages in order to build the Checker
-Framework.  The following command installs them for Ubuntu:
+Framework. The following command installs them for Ubuntu:
 
 \begin{Verbatim}
 sudo apt-get update
 sudo apt-get install --yes ant dia git hevea junit4 librsvg2-bin make mercurial openjdk-8-jdk \
  texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended unzip
 \end{Verbatim}
 
+% Building Checker Framework from source on Mac OS
+
+If you are building on Mac OS, then the following commands install the aforementioned packages:
```

Comment:
I think it would be best to remark that a Mac user should use these commands only if the user already employs homebrew to install packages.  The manual could say that instructions are forthcoming for macports users.